lib/web_console/templates/console.js.erb

```diff
@@ -268,6 +394,7 @@ REPLConsole.prototype.removeCaretFromPrompt = function() {
 };
 
 REPLConsole.prototype.setInput = function(input, caretPos) {
+  if (typeof input === 'undefined') return; // keep value if input is undefined
```

Comment:
Can we drop those undefined checks with `input == null`?  Both `null` and `undefined` don't suit us here, and `input == null` is a bit less verbose. This is one of the few legit reasons to use `==` in JS. :-)